File: leetcode217.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        dic = {}
        for x in nums:
            if x in dic:
                return True
            else:
                dic[x] = 1
                
        return False

File: leetcodec++/leetcode217.cpp
//
// Created by 黄敏辉 on 2019-09-03.
//

class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        set<int> myset;
        for (auto & num:nums){
            if(myset.find(num) != myset.end())
                return true;
            myset.insert(num);
        }
        return false;
    }
};